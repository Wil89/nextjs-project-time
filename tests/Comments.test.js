import Comments from "../components/input/comments";
import { render, cleanup } from "@testing-library/react";

describe("Comments components tests", () => {
  afterEach(cleanup);

  test("should match snapshot", () => {
    const { container } = render(<Comments />);
    expect(container).toMatchSnapshot();
  });
});
