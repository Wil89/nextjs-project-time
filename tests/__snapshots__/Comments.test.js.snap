// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments components tests should match snapshot 1`] = `
<div>
  <section
    class="comments__Comments-sc-1lcdrb2-0 fCdsdL"
  >
    <button
      data-testid="button-toggle"
    >
      Show
       Comments
    </button>
    <ul
      class="comment-list__CommentList-sc-18i8ejw-0 fEXOaQ"
    />
  </section>
</div>
`;
